components/dashboard/src/workspaces/Workspaces.tsx

```diff
@@ -44,139 +45,147 @@ export default function () {
         })();
     }, [teams, location]);
 
+    useEffect(() => {
+        setIsEmpty(!activeWorkspaces.length && !inactiveWorkspaces.length && !workspaceModel?.searchTerm);
+    }, [activeWorkspaces, inactiveWorkspaces, workspaceModel]);
```

Comment:
Would it be esier to read if we had:

```
const hasWorkspaces = activeWorkspaces.length || inactiveWorkspaces.length
```

and below, we would check:

```
{hasWorkspaces ||  workspaceModel?.searchTerm (
```

Reply:
> I think it's the state for this search bar:

I haven't noticed this feature 😂 Updated.

I add a condition `isStartWorkspaceModalVisible`, in case toggle cmd+O to select repo cancel will go with select code (no idea why we need)